vscode-dotnet-runtime-library/src/Acquisition/DotnetCoreAcquisitionWorker.ts

```diff
@@ -27,7 +27,8 @@ export class DotnetCoreAcquisitionWorker {
     private acquisitionPromises: { [version: string]: Promise<string> | undefined };
 
     constructor(private readonly context: IAcquisitionWorkerContext) {
-        this.installDir = path.join(this.context.storagePath, '.dotnet');
+        const installFolderName = process.env.VSCODE_DOTNET_INSTALL_FOLDER || '.dotnet';
```

Comment:
if `VSCODE_DOTNET_INSTALL_FOLDER` is meant to only be there for testing prefix it with `__`.